packages/cms-lib/hubdb.js

```diff
@@ -50,20 +48,40 @@ async function createHubDbTable(portalId, src) {
 
   let response;
   if (rowsToUpdate.length > 0) {
-    response = await createRows(portalId, id, rowsToUpdate);
+    response = await createRows(portalId, tableId, rowsToUpdate);
   }
 
-  await publishTable(portalId, id);
+  await publishTable(portalId, tableId);
 
   return {
-    tableId: id,
+    tableId: tableId,
     rowCount:
       response && Array.isArray(response) && response.length
         ? response[0].rows.length
         : 0,
   };
 }
 
+async function createHubDbTable(portalId, src) {
+  validateJson(src);
+
+  const table = fs.readJsonSync(src);
+  const { rows, ...schema } = table;
+  const { columns, id } = await createTable(portalId, schema);
+
+  return updateRows(portalId, id, rows, columns);
+}
+
+async function importHubDbTableRows(portalId, tableId, src) {
+  validateJson(src);
+
+  const table = fs.readJsonSync(src);
+  const { rows } = table;
+  const { columns } = await fetchTable(portalId, tableId);
+
+  return updateRows(portalId, tableId, rows, columns);
```

Comment:
This code is making an assumption that columns from the table that is fetched match up with the structure of the columns in the table file that rows are being imported from.